packages/tradershub/src/features/cfd/components/MT5PlatformsList/MT5PlatformsList.tsx

```diff
@@ -15,14 +15,15 @@ type TMT5PlatformsListProps = {
 const MT5PlatformsList = ({ onMT5PlatformListLoaded }: TMT5PlatformsListProps) => {
     const { isFetching } = useAuthorize();
     const { uiState } = useUIContext();
+    const { accountType } = uiState;
```

Comment:
```suggestion
    const { accountType, regulation } = uiState;
    const activeRegulation = regulation;
```
if you are going to use destructure approach then you have to get regulation from it too and replace line#19

Reply:
@hamza-deriv  I was going to do it somehow discarded it with other changes thanks for pointing it out 🙇‍♀️